src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,13 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
import static should come before normal imports (at least according to checkstyle.xml) 
The order rule "STATIC###STANDARD_JAVA_PACKAGE###SPECIAL_IMPORTS###THIRD_PARTY_PACKAGE" consists of:
      1. STATIC: static imports
      2. STANDARD_JAVA_PACKAGE: standard java/javax imports